src/shared/tpm2-util.c

```diff
@@ -732,6 +732,17 @@ size_t tpm2_tpml_pcr_selection_weight(const TPML_PCR_SELECTION *l) {
         return weight;
 }
 
+static void tpm2_log_debug_tpml_pcr_selection(const TPML_PCR_SELECTION *l, const char *msg) {
+        if (!DEBUG_LOGGING || !l)
+                return;
+
+        _cleanup_free_ char *s = tpm2_tpml_pcr_selection_to_string(l);
+        if (s)
+                log_debug("%s: %s", msg ?: "PCR selection", s);
+        else
+                log_oom();
```

Comment:
Same here.
```c
log_debug("%s: %s", msg ?: "PCR selection", strna(s));
```
should be enough. If you really want to log about OOM, then please use `log_debug_oom()`.

Reply:
ack